packages/utilities/psammead-locales/moment/index.stories.jsx

```diff
@@ -61,11 +62,13 @@ const locales = [
   { name: 'Marathi', locale: 'mr' },
   { name: 'Mundo', locale: 'es' },
   { name: 'Nepali', locale: 'ne' },
+  { name: 'News', locale: 'en-GB' },
```

Comment:
It might be best for the casing to match that of the locale: `en-gb`.

Reply:
Very good point.  Wil fix